build-system/tasks/visual-diff/helpers.js

```diff
@@ -65,61 +65,50 @@ function log(mode, ...messages) {
  *
  * @param {!puppeteer.Page} page a Puppeteer control browser tab/page.
  * @param {string} testName the full name of the test.
- * @param {!Array<string>} forbiddenCss Array of CSS elements that must not be
- *     found in the page.
- * @param {!Array<string>} loadingIncompleteCss Array of CSS elements that must
- *     eventually be removed from the page.
- * @param {!Array<string>} loadingCompleteCss Array of CSS elements that must
- *     eventually appear on the page.
- * @throws {string} an encountered error.
+ * @param {!Array<string>} selectors Array of CSS selector that must eventually
+ *     be removed from the page.
+ * @throws {Error} an encountered error.
  */
-async function verifyCssElements(page, testName, forbiddenCss,
-  loadingIncompleteCss, loadingCompleteCss) {
-  // Begin by waiting for all loader dots to disappear.
-  if (!(await waitForLoaderDot(page))) {
-    throw new Error(`${colors.cyan(testName)} still has the AMP loader dot ` +
-        `after ${CSS_SELECTOR_TIMEOUT_MS} ms`);
-  }
-
-  if (forbiddenCss) {
-    for (const css of forbiddenCss) {
-      if ((await page.$(css)) !== null) {
-        throw new Error(`${colors.cyan(testName)} | The forbidden CSS ` +
-            `element ${colors.cyan(css)} exists in the page`);
-      }
-    }
-  }
-
-  if (loadingIncompleteCss) {
+async function verifySelectorInvisible(page, testName, selectors) {
```

Comment:
nit: Pluralize to `verifySelectorsInvisible`, since more than one selector can be passed in.

Reply:
Done